### PR TITLE
Updating info on Windows version support

### PR DESCRIPTION
**Pull request by sfshaza2**
At @timsneath's request.

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]
Ptal @cbracken

File: src/get-started/install/windows.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ next:
 To install and run Flutter,
 your development environment must meet these minimum requirements:
 
-- **Operating Systems**: Windows 7 SP1 or later (64-bit), x86-64 based.
+- **Operating Systems**: Windows 10 SP1 or later (64-bit), x86-64 based.
```
Comment (timsneath) [inline src/get-started/install/windows.md:17]: Windows 10 or later (not Windows 10 SP1 or later).

**Review by timsneath — CHANGES_REQUESTED** [review]
Not SP1 :)

**Review by domesticmouse — APPROVED** [review]

**Comment by domesticmouse** [comment]
Looks like the snippets need to be refreshed

**Review by cbracken — APPROVED** [review]
lgtm; thanks!

**Comment by sfshaza2** [comment]
@timsneath can you approve the changes? Only the person who requested changes can approve them.

**Review by timsneath — APPROVED** [review]

**Comment by sfshaza2** [comment]
Thx, @timsneath!

**Pull request merged**